.github/labels.yml

```
@@ -29,30 +29,33 @@
 - name: enhancement
   description: adds a new feature or improves on an existing one
   color: ad12cc
-- name: dependencies
-  description: related to dependencies
+- name: 'bump: major'
+  description: A major breaking change
   color: fcf3b8
-- name: major
-  description: Increment the major version when merged
-  color: c5000b
-- name: minor
-  description: Increment the minor version when merged
-  color: b54f0b
-- name: patch
-  description: Increment the patch version when merged
-  color: 067314
-- name: release
-  description: Create a release when this pr is merged
-  color: 38e0d5
-- name: internal
-  description: Changes only affect the internal API
-  color: 696969
-- name: documentation
+- name: 'bump: feature'
```

> Maybe there is a better prefix that encompasses both cases? - me

I meant for the chore ones and these.
```suggestion
- name: 'change: feature'
```

---

Sorry, I misunderstood.  I've changed them all to `change`.  I've made the label colour different for the two that don't make a release.